Day-I-Social-Card/social-card/src/App.js

```diff
@@ -1,23 +1,21 @@
 import React from 'react';
 import './App.css';
+import HeaderContainer from './components/HeaderComponents/HeaderContainer';
+import CardContainer from './components/CardComponents/CardContainer'
+import Footer from './components/FooterComponents/Footer'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faComment,faSyncAlt,faHeart, faEnvelope,faComments} from '@fortawesome/free-solid-svg-icons'
+
+library.add(faComment, faSyncAlt, faHeart, faEnvelope,faComments)
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Social Card!</h3>
-      <p>
-        Begin by exploring the `components` directory. You'll notice we have a
-        few files that we've already included in there to get you started right
-        away building components. You'll need to make sure you include your
-        components that you build in this file to watch your app come to life
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
+    <div className='appContainer'>
+   <HeaderContainer/>
```

Comment:
This should still follow the same html spacing. The children should be properly spaced under the parent.